gen/generate.go

```diff
@@ -368,8 +374,9 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 	jsonData := protocol.EncodeJSON(g)
 	err = ioutil.WriteFile(filepath.Join(outDir, config.GenesisJSONFile), append(jsonData, '\n'), 0666)
 
-	if (!verbose) && (rootKeyCreated > 0 || partKeyCreated > 0) {
+	if (verbose) && (rootKeyCreated > 0 || partKeyCreated > 0) {
```

Comment:
In what way? Do you think this output should be written either way? I was under the assumption that this `verbose` flag could have been `!silent` all the same, since it is checked before writing any other output in this function.